charts/apps/kubeflow-pipelines/s3-only-static/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml

```diff
@@ -0,0 +1,254 @@
+apiVersion: v1
+data:
+  sync.py: "# Copyright 2020-2021 The Kubeflow Authors\n#\n# Licensed under the Apache\
```

Comment:
grab a more readable version from the kustomized.yaml from `tools/helmify/generated_output/kustomized_output_files`

Reply:
same as previous comment